docs/onboard/2_data.md

```diff
@@ -27,7 +27,7 @@ The caveman graph consists of M clusters each with N nodes, every cluster has a
 
 ### Save
 
-Then we save the graph in [json format](~/advanced/data_spec.md) which includes files `graph.json` and `meta.json`. `graph.json` contains all feature and connectivity information. `meta.json` describes the high level information of the graph: the number of node and edge types and the number of typed attributes for nodes and edges.
+Then we save the graph in [json format](~/advanced/data_spec.md) which includes the file `graph.json`. `graph.json` contains all feature and connectivity information.
```

Comment:
Remove stutter, 
`graph.json` contains all feature and connectivity information. -> It contains all feature and connectivity information.